quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/ShowConfig.java

```diff
@@ -70,8 +71,8 @@ public void run() {
                         .forEachOrdered(this::printProperty);
             }
 
-            if (!parseBoolean(System.getProperty("kc.show.config.runtime", Boolean.FALSE.toString()))) {
-                System.exit(0);
+            if (!Boolean.getBoolean("kc.show.config.runtime")) {
```

Comment:
Should we load all the `System.properties` in one place? e.g. `Environment.java`

Reply:
I would say this one is very specific to the `show-config` command and not something useful outside this scope.